AwsCryptographicMaterialProviders/runtimes/python/pyproject.toml

```diff
@@ -1,28 +1,52 @@
 [tool.poetry]
 name = "aws-cryptographic-materialproviders"
-version = "0.1.0"
-description = ""
+version = "0.1.1"
+description = "AWS Cryptographic Material Providers Library for Python"
 authors = ["AWS Crypto Tools <aws-crypto-tools@amazon.com>"]
 packages = [
     { include = "aws_cryptographic_materialproviders", from = "src" },
 ]
 # Include generated internaldafny files in package distributions,
 # even though they are not included in version control
 include = ["**/internaldafny/generated/*.py"]
+readme = "README.rst"
 
 [tool.poetry.dependencies]
 python = "^3.11.0"
-# TODO: Bump this to 4.8.0
-# TODO: Longer-term, write something to pull this in from the project's project.properties file
-DafnyRuntimePython = "~4.7.0"
-aws-cryptography-internal-standard-library = { path = "../../../StandardLibrary/runtimes/python", develop = false}
-aws-cryptography-internal-kms = {path = "../../../ComAmazonawsKms/runtimes/python", develop = false}
-aws-cryptography-internal-dynamodb = {path = "../../../ComAmazonawsDynamodb/runtimes/python", develop = false}
-aws-cryptography-internal-primitives = {path = "../../../AwsCryptographyPrimitives/runtimes/python", develop = false}
+aws-cryptography-internal-standard-library = {path = "../../../StandardLibrary/runtimes/python"}
+aws-cryptography-internal-kms = {path = "../../../ComAmazonawsKms/runtimes/python"}
+aws-cryptography-internal-dynamodb = {path = "../../../ComAmazonawsDynamodb/runtimes/python"}
+aws-cryptography-internal-primitives = {path = "../../../AwsCryptographyPrimitives/runtimes/python"}
+
+# Package testing
+
+[tool.poetry.group.test]
+optional = true
 
 [tool.poetry.group.test.dependencies]
 pytest = "^7.4.0"
 
+# Package release
+
+[tool.poetry.group.release]
+optional = true
+
+[tool.poetry.group.release.dependencies]
+poetry = "1.8.3"
```

Comment:
comment: will these deps be in scope for dependabot updates?

Reply:
I think so, with the same priority as dev dependencies.